Task10/App/Entities/YoutubeItem.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace App.Entities
+{
+    public class YoutubeItem
+    {
+        public string ImageUrlMedium;
+        public string ImageUrlHigh;
+        public string ImageUrlDefault;
+        public string Title;
+        public string Description;
+        public string Id;
+
+        public YoutubeItem(string imageUrlMedium,
```

Comment:
I don't think that this constructor with such long list of arguments makes sense. Instead of, you can use :
```
new YoutubeItem
{
    ImageUrlMedium = ...,
    ImageUrlHigh = ...
}
```

Reply:
Ok, done